doc/programming_guide/sprites/spritelists.rst

```diff
@@ -0,0 +1,115 @@
+.. _pg_spritelists:
+
+What's a Sprite?
+----------------
+
+Each sprite describes where a game object is & how to draw it. This includes:
+
+* Where it is in the world
+* Where to find the image data
+* How big the image should be
+
+The rest of this page will explain using the ``SpriteList`` class to draw
+sprites to the screen.
+
+
+.. _pg_spritelists_why:
+
+Why SpriteLists?
+----------------
+
+.. _pg_spritelists_why_hardware:
+
+They're How Hardware Works
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Graphics hardware is designed to draw groups of objects at the same time.
+These groups are called **batches**.
+
+Each :py:class:`~arcade.SpriteList`
+automatically translates every :py:class:`~arcade.Sprite` in it
+into an optimized batch. It doesn't matter if a batch has one or hundreds of
+sprites: it still takes the same amount of time to draw!
+
+This means that using fewer batches helps your game run faster, and that you
+should avoid trying to draw sprites one at a time.
+
+
+.. _pg_spritelists_why_faster_dev:
+
+They Help Develop Games Faster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sprite lists do more than just draw. They also have built-in features which save
+you time & effort, including:
+
+* Automatically skipping off-screen sprites
+* Collision detection
+* Debug drawing for hit boxes
+
+
+.. _pg_spritelists_minimal_sprite_drawing:
+
+Drawing with Sprites and SpriteLists
+------------------------------------
+
+Let's get to the example code.
+
+There are 3 steps to drawing sprites with a sprite list:
+
+1. Create a :py:class:`~arcade.SpriteList`
+2. Create & append your :py:class:`~arcade.Sprite` instance(s) to the list
+3. Call :py:meth:`~arcade.SpriteList.draw` on your SpriteList inside an
+   :py:meth:`~arcade.Window.on_draw` method
+
+Here's a minimal example:
+
+.. code:: python
+
+    import arcade
+
+
+    class WhiteSpriteCircleExample(arcade.Window):
+
+        def __init__(self):
+            super().__init__(800, 600, "White SpriteCircle Example")
+            self.sprites = None
+            self.setup()
+
+        def setup(self):
+            # 1. Create the SpriteList
+            self.sprites = arcade.SpriteList()
+
+            # 2. Create & append your Sprite instance to the SpriteList
+            self.circle = arcade.SpriteCircle(30, arcade.color.WHITE)  # 30 pixel radius circle
+            self.circle.position = self.width // 2, self.height // 2  # Put it in the middle
+            self.sprites.append(self.circle)  # Append the instance to the SpriteList
+
+        def on_draw(self):
+            # 3. Call draw() on the SpriteList inside an on_draw() method
+            self.sprites.draw()
+
+
+    game = WhiteSpriteCircleExample()
+    game.run()
```

Comment:
Might think of pulling this out as a separate file? Then the file could be run as part of the examples test?